README.md

```diff
@@ -1,11 +1,7 @@
-<!-- marker-before-logo -->
```

Comment:
Could you leave the markers in there? They are important for the sphinx docs.

Reply:
I just saw that don't make use of this header anyways in docs/index.rst, so all good :+1: